src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -1120,6 +1152,40 @@ def libfuzzerize_corpus(self):
 
     return new_units_generated, new_units_added, corpus_size
 
+  def generate_new_testcase_mutations(self):
+    """Generate new testcase mutations using radamsa, etc."""
+    target_name = os.path.basename(self.target_path)
+    project_qualified_fuzzer_name = (
+        data_types.fuzz_target_project_qualified_name(utils.current_project(),
+                                                      target_name))
+
+    generation_timeout = engine_common.get_new_testcase_mutations_timeout()
+    old_input_directory_file_count = shell.get_directory_file_count(
+        self.afl_input.input_directory)
+
+    # Generate new testcase mutations using Radamsa.
+    if self.strategies.generator == engine_common.Generator.RADAMSA:
+      expected_completion_time = time.time() + generation_timeout
+      engine_common.generate_new_testcase_mutations_using_radamsa(
+          self.afl_input.input_directory, self.afl_input.input_directory,
+          expected_completion_time)
```

Comment:
since you are here, can you things more consistent by making generate_new_testcase_mutations_using_radamsa take in a timeout rather than an expected completion time?